tests/kustomize/runner/resources/image_referencer/overlays/prod/kustomization.yaml

```diff
@@ -1,3 +1,6 @@
 resources:
--   ../../base
-namePrefix: prod-
+- ../../base
+
+images:
```

Comment:
why is the kustomize changed?

Reply:
wanted to test, if this just an issue with the name, but it seems to be a general issue with kustomize. marked it as `xfail` for Python 3.9